airflow/www/views.py

```diff
@@ -3012,6 +3015,57 @@ def action_muldelete(self, items):
         self.update_redirect()
         return redirect(self.get_redirect())
 
+    @action(
+        'mulduplicate',
+        'Duplicate',
+        'Are you sure you want to duplicate the selected connections?',
+        single=False,
+    )
+    @provide_session
+    @auth.has_access(
+        [
+            (permissions.ACTION_CAN_CREATE, permissions.RESOURCE_CONNECTION),
+        ]
+    )
+    def action_mulduplicate(self, connections, session=None):
+        """Duplicate Multiple connections"""
+        for selected_conn in connections:
+            new_conn_id = selected_conn.conn_id
+            matches = re.findall(r"_copy(\d+)$", selected_conn.conn_id)
```

Comment:
Since there can only be one match here, it’s probably better to do

```python
match = re.search(...)
if match is not None:
    ...
```

instead.

Reply:
yeah, agree
**re.search()** will also match the last occurence.
using that.